core/promise/methods/noop/promise_processor.go

```diff
@@ -63,12 +69,13 @@ type PromiseProcessor struct {
 }
 
 // Start processing promises for given service proposal
-func (processor *PromiseProcessor) Start(proposal discovery_dto.ServiceProposal) error {
+func (processor *PromiseProcessor) Start(proposal dto.ServiceProposal) error {
 	processor.lastPromise = promise.Promise{
-		Amount: money.NewMoney(10, money.CURRENCY_MYST),
+		Fee: money.NewMoney(10, money.CURRENCY_MYST),
```

Comment:
Why '10'? We prob should externalise this one.

Reply:
I think will be improved in upcoming PRs. TODO needed